source/administration/config-settings.rst

```diff
@@ -4037,6 +4037,102 @@ Changes to this setting require a server restart before taking effect.
 Experimental Settings only in config.json
 -----------------------------------------
 
+Audit settings
+~~~~~~~~~~~~~~~
+The audit settings output audit records to syslog (local or remote server via TLS) and/or to a local file. Both are disabled by default.
+
+Syslog configuration options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: When ``true`` syslog output is enabled.
+**False**: Syslog output is disabled.
+
++-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsSysLogEnabled": "false"`` with options ``true`` and ``false``.                              |
```

Comment:
I don't think the section and key should be combined here.  The section is `ExperimentalAuditSettings` and the key is `ExperimentalAuditSettingsSysLogEnabled`.  Also, the default value is a bool not a string.  e.g. ``"SysLogEnabled": false``

Reply:
Updated, thanks!